pmm/pmm2-ami-staging-start.groovy

```diff
@@ -175,7 +187,11 @@ pipeline {
                             set -o errexit
                             set -o xtrace
                             [ ! -d "/home/centos" ] && echo "Home directory for centos user does not exist"
-                            sudo yum -y install git svn sysbench
+                            sudo yum -y install git svn docker
+                            sudo systemctl start docker
+                            sudo curl -L https://github.com/docker/compose/releases/download/1.29.0/docker-compose-`uname -s`-`uname -m` | sudo tee /usr/bin/docker-compose > /dev/null
```

Comment:
it is not good idea to download and execute any binaries without at least checking md5

Reply:
I have added the check.